install/helm/agones/values.yaml

```diff
@@ -216,6 +216,9 @@ agones:
     controller:
       name: agones-controller
       pullPolicy: IfNotPresent
+    # extensions settings ignored unless `SplitControllerAndExtensions` feature gate is enabled
+    extensions:
+      name: agones-extensions
```

Comment:
Just making a note that we had an offline discussion that for now - values would get pulled from the `controller` configuration section.

If we wanted to move an extra single value out (not for this PR), then `replicas` would probably be useful. That being said, as we split this out further, I do wonder how much we end up with that is duplicate between the controller and extensions. 🤔